formats/topical_event_about_page/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "parent"
+  ],
+  "properties": {
+    "parent": {
+      "$ref": "#/definitions/guid_list"
```

Comment:
In fact, `parent` is already defined in `base_links.json` with `"maxItems": 1`, so we don't need to include it here. I'll rebase shortly.